test/jest/NodeKey.spec.ts

```diff
@@ -60,4 +61,21 @@ describe('NodeKey', () => {
     const nodeKey = NodeKey.fromBytes(await randomBytes(22));
     validateNodeKey(nodeKey);
   });
+
+  test('it should derive a child seed from private key', async () => {
+    const expectedChildSeed = 'crouch cause bicycle tomato choice city produce ocean canyon rare laugh sudden close angle cage item special jeans either heavy skill bracket lift butter';
+    const nodeKey = NodeKey.fromBytes(
+      Buffer.from([
+        0x60, 0x6c, 0x9b, 0xfe,
```

Comment:
Where are these bytes from? If it's an example test published somewhere maybe we can link to it in the comments. If it's just arbitrary maybe we can use a snapshot? It's just not very clear what this is testing.

Reply:
It's arbitrary bytes to ensure that we get the same child seed given this static master private key. I changed the assertion below to use a snapshot.

Reply:
This also ensures that we get a failing test if we change the order of `SwapClientType` enum.